python/singa/autograd.py

```diff
@@ -358,6 +358,48 @@ def relu(x):
     return ReLU()(x)[0]
 
 
+class Elu(Operation):
+    def __init__(self,alpha=1):
+        super(Elu, self).__init__()
+        self.alpha=alpha
+
+    def forward(self, x):
+        """Do forward propgation.
+        Store the x if requires gradient.
+        Args:
+            x (CTensor): matrix
+        Returns:
+            a CTensor for the result
+        """
+        #f(x) = alpha * (exp(x) - 1.) for x < 0, f(x) = x for x >= 0
+        if training:
+            self.input = x
+        x1 = singa.LTFloat(x, 0.0)
+        x1 = singa.__mul__(x, x1)
+        x1 = singa.SubFloat(singa.Exp(x1),self.alpha)
```

Comment:
I think this formula doesn't correspond to alpha * (exp(x) - 1.)

Reply:
sorry for my fault, I just modified